File: src/test/resources/ib_action_test.m
@implementation IBActionTest

- (IBAction)telButtonTapped:(id)sender {
    [self.delegate telButtonTappedForCell:self];
}

- (IBAction)queryButtonTapped:(id)sender {
    if ([self.delegate respondsToSelector:@selector(queryTapped:forBookmarkCell:)]) {
        [self.delegate queryTapped:self.entity.query forBookmarkCell:self];
    }
}

@end

File: src/test/resources/ib_action_test.swift
class IBActionTest {

    @IBAction func telButtonTapped(sender: AnyObject) {
        self.delegate.telButtonTappedForCell(self)

    }

    @IBAction func queryButtonTapped(sender: AnyObject) {
        if self.delegate.respondsToSelector("queryTapped:forBookmarkCell:") {
            self.delegate.queryTapped(self.entity.query, forBookmarkCell: self)
        }

    }

}
